### ISSUE TITLE
importing @grafana/faro-web-tracing breaks nuxt app

### ISSUE THREAD

**Issue by DanielYatali** [grafana/faro-web-sdk#609]
## Description
I have a pretty simple plugin setup for grafana using @grafana/faro-web-sdk

``` javascript
import {getWebInstrumentations, initializeFaro} from '@grafana/faro-web-sdk';
export default defineNuxtPlugin(nuxtApp => {
    const config = useRuntimeConfig()

    let instrumentation = [...getWebInstrumentations()]
    const faro = initializeFaro({
        url: config.public.grafanaUrl,
        instrumentations: instrumentation,
        app: {
            name: "test-app",
            version: config.public.version,
            environment: config.public.environment
        },
    });
    console.log('Faro initialized')
    nuxtApp.provide('faro', faro)
})
```
When trying to implement tracing however.

``` javascript 
import {getWebInstrumentations, initializeFaro} from '@grafana/faro-web-sdk';
import {TracingInstrumentation} from '@grafana/faro-web-tracing';

export default defineNuxtPlugin(nuxtApp => {
    const config = useRuntimeConfig()

    let instrumentation = [...getWebInstrumentations()]
    instrumentation.push(new TracingInstrumentation({
        instrumentationOptions: {
            propagateTraceHeaderCorsUrls: [new RegExp('https://example/*'), new RegExp('https://test-example/*')]
        }
    }))
    const faro = initializeFaro({
        url: config.public.grafanaUrl,
        instrumentations: instrumentation,
        app: {
            name: "test-app",
            version: config.public.version,
            environment: config.public.environment
        },
    });
    console.log('Faro initialized')
    nuxtApp.provide('faro', faro)
})
```
The app breaks, see error message:
![image](https://github.com/grafana/faro-web-sdk/assets/70588497/dd7c59e3-23e7-4915-80ed-551e36f4bdd5)

I removed the instrumentation code and left the import for @grafana/faro-web-tracing and although I'm not using it, the app returns the same error message

## Steps to reproduce

Setup fresh nuxt app https://nuxt.com/docs/getting-started/installation
Install @grafana/faro-web-sdk  and @grafana/faro-web-tracing
Create plugin inside the plugins directory called grafana.js and copy the code above.

## Environment

- **SDK version:** 1.7.3
- **SDK instrumentations:** Web SDK, Web Tracing
- **Device type:** desktop
- **OS:** Windows
- **Browser:** Chrome
- **Nuxt version: ** 3.11.2

**Comment by codecapitano** [comment]
Hi @DanielYatali 

It's working for me without any issues.

Only thing I changed is to rename `grafana.js` to `grafana.client.js`.
This is because Faro is client side only and does not work on the server.
So have to tell Nuxt to load the plugin only in the client.

For that you can use a `.client` suffix in the file name to load a plugin on client only. See [docs for the plugins directory](https://nuxt.com/docs/guide/directory-structure/plugins).

TLDR: 
<img width="795" alt="image" src="https://github.com/grafana/faro-web-sdk/assets/47627413/ff53f6c8-2533-48e8-a516-9d4b9c449de2">

Cheers

**Comment by DanielYatali** [comment]
We only use client-side in our project, so we set `ssr=false` in our Nuxt.js configuration.
After some debugging found the problem.
When the `pages` directory is introduced (which is essential for routing between pages) it throws that error. 
I pinpointed the problem to a new feature introduced in Nuxt.js version [3.8.0](https://github.com/nuxt/nuxt/releases/tag/v3.8.0) specifically the `App Manifest` feature. To resolve the issue, I followed the Nuxt documentation and disabled this feature in our configuration:

```javascript
export default defineNuxtConfig({
  experimental: {
    appManifest: false
  }
});
```

Disabling the `App Manifest` resolved the error.

**Comment by codecapitano** [comment]
Awesome thanks so much for the update @DanielYatali

**Comment by Gazoon007** [comment]
> Hi [@DanielYatali](https://github.com/DanielYatali)
> 
> It's working for me without any issues.
> 
> Only thing I changed is to rename `grafana.js` to `grafana.client.js`. This is because Faro is client side only and does not work on the server. So have to tell Nuxt to load the plugin only in the client.
> 
> For that you can use a `.client` suffix in the file name to load a plugin on client only. See [docs for the plugins directory](https://nuxt.com/docs/guide/directory-structure/plugins).
> 
> TLDR: <img alt="image" width="795" src="https://private-user-images.githubusercontent.com/47627413/338197032-ff53f6c8-2533-48e8-a516-9d4b9c449de2.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3Mjc3NDc2NzksIm5iZiI6MTcyNzc0NzM3OSwicGF0aCI6Ii80NzYyNzQxMy8zMzgxOTcwMzItZmY1M2Y2YzgtMjUzMy00OGU4LWE1MTYtOWQ0YjljNDQ5ZGUyLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDEwMDElMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMDAxVDAxNDkzOVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWQ5NTNkNmQ3MWI0MTE5NDNlNzQ1Mjk2ZWE1Mjk4M2Q4ZTMxZWY2MTYyZTVhNmY0MmI2ZTY3YzYxNThkMDgzYzMmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.YEGyzJMSReY8mtn0VBo2UTO9m4mpGa6MjoqRaNHyZEI">
> 
> Cheers

Genuinely asking, Grafana Faro can't log any error that happened on server-side, so it is pointless for me to use Faro if my project using SSR?
